data-model/fdo-profiles/0.1.0/source-system/schema/source-system-request.json

```diff
@@ -0,0 +1,56 @@
+{
+  "$id": "https://schemas.dissco.tech/schemas/fdo-profiles/0.1.0/source-system-request.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "type": "object",
+  "description": "Kernel from which all DiSSCo FAIR Digital Object profiles are derived",
+  "$comment":"FDO Profile Version 0.1.0",
+  "properties": {
+    "fdoProfile" : {
+      "type": "string",
+      "description": "PID to a machine readable description of the attributes in the FDO record",
+      "example": "https://hdl.handle.net/21.T11148/d8de0819e144e4096645",
+      "$comment": "idx = 1"
+    },
+    "digitalObjectType": {
```

Comment:
Shouldn't this be fdoType?

Reply:
No, this is correct. It's based on the handle profile, which calls this field "digitalObjectType"